dpx/src/dpx_type1c.rs

```diff
@@ -347,15 +329,7 @@ pub unsafe fn pdf_font_load_type1c(font: *mut pdf_font) -> i32 {
     if cffont.flag & 1i32 << 0i32 != 0 {
         panic!("This is CIDFont...");
     }
-    let fullname =
-        new((strlen(fontname).wrapping_add(8)).wrapping_mul(::std::mem::size_of::<i8>()) as _)
-            as *mut i8;
-    sprintf(
-        fullname,
-        b"%6s+%s\x00" as *const u8 as *const i8,
-        uniqueTag,
-        fontname,
-    );
+    let fullname = format!("{}+{}", uniqueTag, font.fontname);
```

Comment:
"{:>6}+{}"

Reply:
uniqueTag is always 6 characters long. `"{}+{}"` looks a bit cleaner.